src/FNAPlatform/SDL2_FNAPlatform.cs

```diff
@@ -970,6 +1019,10 @@ public static void RunLoop(Game game)
 				}
 
 				Keyboard.SetKeys(keys);
+				if (TouchDeviceExists)
```

Comment:
I just realized: This part may actually need to go into FrameworkDispatcher instead, for accuracy:

https://github.com/FNA-XNA/FNA/blob/master/src/FrameworkDispatcher.cs#L31

Functionally it's not a big deal for this one block, but the catch is that `TouchDeviceExists` may have to become an internal variable in TouchPanel or something... the init process can be the same, it's just this one update that has to move.